sql_metadb/derived_tables/runlist.txt

```diff
@@ -26,6 +26,7 @@ instance_formats.sql
 instance_editions.sql
 instance_languages.sql
 instance_nature_content.sql
+instance_administrative_notes.sql
```

Comment:
There are no dependencies on other derived tables. So it should be sorted alphabetically.

Reply:
I wasn't aware of this convention @stdombek. I will make the change.

Reply:
I'm not sure if we have such a convention.  At the moment I can't think what the advantage would be of sorting only some elements.

Reply:
As long as there are no dependencies, it would be better to sort alphabetically. If further derived tables are added later that are dependent, it is easier to sort them in.